src/App.jsx

```diff
@@ -1,7 +1,41 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import WeatherRequest from './WeatherRequest';
+
+const API_KEY = '39023aff7ae25b6655f72f0935e31fc6';
 
 function App() {
+  const [currentWeather, setCurrentWeather] = useState();
+  const [hourlyWeather, setHourlyWeather] = useState();
+  const [dailyWeather, setDailyWeather] = useState();
+
+  const addCurrentWeather = function addCurrentWeatherFromData(weatherData) {
+    setCurrentWeather(weatherData);
+  };
+  const addHourlyWeather = function addHourlyWeatherFromData(weatherData) {
+    setHourlyWeather(weatherData);
+  };
+  const addDailyWeather = function addDailyWeatherFromData(weatherData) {
+    setDailyWeather(weatherData);
+  };
+
+  const coordinates = {
+    latitude: '45.50742',
+    longitude: '-122.68984',
+  };
+
+  WeatherRequest({
```

Comment:
Is WeatherRequest a component or function? I thought to render a component, the syntax was `<WeatherRequest />`

Reply:
We can discuss this later but the useEffect inside of a function call can be a bit tricky. Unless we change this to a component, the useEffect won't trigger on the change in its dependencies since it is a function call.